scripts/run_tests_coverage

```diff
@@ -6,7 +6,7 @@ source "$SCRIPTS_DIR/common"
 
 # Build the example client and Application in coverage mode and run them.
 export RUST_LOG=trace
-"${SCRIPTS_DIR}/xtask" --scope=all run-examples --example-name=abitest --server-variant=coverage
+"${SCRIPTS_DIR}/xtask" --scope=all run-oak-functions-examples --example-name=abitest --server-variant=coverage
```

Comment:
This does not look right. We don't have an oak-fuctions "abitest" example.
Also, for oak-functions we don't have a server-variant for "coverage".

Reply:
That is true, that seems to be a left-over of Oak classic. 

So we should probably just remove this call? 

(IIUC it does currently simply nothing because xtask will not find an example with this name---correction, just tried: it fails because we don't have the server-variant "coverage".)